v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm. 
```

Comment:
I am not seeing any lines with spaces at the end of lines. I even cloned the repo and edited the file and could not find a lines with spaces at the end.